src/test/scala/nl.knaw.dans.easy.bagstore/component/BagStoresSpec.scala

```diff
@@ -99,10 +99,41 @@ class BagStoresSpec extends TestSupportFixture
     }
   }
 
+  it should "result in a failure when a get is done on a hidden bag with forceInactive=false" in {
+    val (bagId: BagId, output: BagPath) = setupTestDeactivateBag
+    // without force option results in failure
+    bagStore1.copyToDirectory(bagId, output.resolve("a1"), false, false) should matchPattern {
+      case Failure(_: InactiveException) =>
+    }
+    // with force option results bagId a success
+    bagStore1.copyToDirectory(bagId, output.resolve("a1"), false, true) should matchPattern {
```

Comment:
all these `should matchPattern { case Success(_) => }` can be replaced by `shouldBe a[Succes[_]]`

Reply:
now i feel dumb